### PR TITLE
fix(db): Rename `message_id` to `_id` 

### PR DESCRIPTION
**Pull request by grtlr**

### CODE REVIEW

**Review by pvdrz — DISMISSED** [review]

**Review by Alex6323 — DISMISSED** [review]

File: bin/inx-chronicle/src/api/responses.rs [hunk 1]
```diff
@@ -34,8 +34,16 @@ pub struct InfoResponse {
 
 impl_success_response!(InfoResponse);
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
-pub struct SyncDataResponse(pub SyncData);
+/// An aggregation type that represents the ranges of completed milestones and gaps.
```
Comment (Alexandcoats) [inline bin/inx-chronicle/src/api/responses.rs:37]: This is needed for the syncer, so relegating it to the API is no bueno

Comment (grtlr) [reply bin/inx-chronicle/src/api/responses.rs:37]: I think this is resolved for now :trollface:

Comment (Alexandcoats) [reply bin/inx-chronicle/src/api/responses.rs:37]: No, we still need it in the syncer

**Comment by grtlr** [comment]
I went ahead and merged this, we can add move it later again if we need to, but my goal is to merge #163 too.

**Pull request merged**